Src/Base/AMReX_GpuDevice.cpp

```diff
@@ -740,6 +744,12 @@ Device::mem_advise_set_preferred (void* p, const std::size_t sz, const int devic
     {
         AMREX_CUDA_SAFE_CALL(cudaMemAdvise(p, sz, cudaMemAdviseSetPreferredLocation, device));
     }
+#elif defined(AMREX_USE_DPCPP)
+    if (device_prop.managedMemory == 1 && device_prop.concurrentManagedAccess == 1)
+    {
+        auto& q = streamQueue();
+        q.mem_advise(p, sz, PI_MEM_ADVICE_SET_PREFERRED_LOCATION);
```

Comment:
When I compile this, I get,
```
error: no member named 'mem_advise' in 'cl::sycl::ordered_queue'
        q.mem_advise(p, sz, PI_MEM_ADVICE_SET_PREFERRED_LOCATION);
```

Reply:
I believe this error is because of this issue https://github.com/intel/llvm/pull/1035. Taking a look at it.

Reply:
It seems that the `in_order` property is not available in beta5 yet.  Otherwise we could switch to use `queue` with `in_order` property.

Reply:
Yes and it is also not available in any internal beta versions yet. Do you suggest to comment out the `mem_advise` for DPCPP block for time being. As with `queue q{property::queue::in_order()};` we will also have the support for `mem_advise` in the future. Please let me know.

Reply:
That sounds good.  Please comment out the `mem_advise` code for now.  Please also add a comment with words "xxxxx DPCPP todo".  I use that for grep.